proposals/umoci.md

```diff
@@ -0,0 +1,101 @@
+The following proposal outlines how this would be achieved.
+
+## Proposal ##
+Change the ownership of the existing umoci project from openSUSE:
+
+  https://github.com/openSUSE/umoci
+
+And move it inside the `opencontainers` organisation:
+
+  https://github.com/opencontainers/umoci
+
+The import paths will correspondingly be "github.com/opencontainers/umoci" (umoci does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+In future we may opt to use vanity imports (such as "umo.ci/cmd/umoci").
+
+In addition, a sub-tree merge (to preserve the history) of the existing image-tools repository
+
+  https://github.com/opencontainers/image-tools
+
+will be done into umoci.
+This will be done to allow the archival of the oci-image-tools repository, as the primary features it has over umoci (specification validation) will be maintained within umoci instead.
+The purpose of this change is to avoid confusion as to which OCI image tool the OCI is shepherding.
+
+The project's domain "umo.ci" could also be transferred to the Linux Foundation so that it can be managed by someone other than the maintainers (though maintainers should still have necessary administrative access to maintain the website).
+
+### Initial Maintainers ###
+Initial maintainers of the umoci project would be:
+
+* Aleksa Sarai <cyphar@cyphar.com> (@cyphar)
+* Tycho Andersen <tycho@tycho.ws> (@tych0)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+It should be noted that since umoci is not a specification, it is not bound by the ordinary quorum and voting rules for specification release.
+As such, new versions will be released as regularly as needed without the need for a quorum vote.
+This is to avoid the serious delays in releases encountered within the `runc` project.
+
+Until there are enough additional maintainers, PRs will be merged with only one maintainer LGTM required.
+Maintainers are encouraged to not merge their own PRs immediately (preferably waiting at least 24 to 48 hours to allow another maintainer to review it), but this is not a strict rule.
+
+### Project Communications ###
+The proposed project would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
```

Comment:
I think this is on freenode (at least, I idle in one called #opencontainers on freenode :)), but probably worth mentioning that here.

Reply:
Will do, though I did just copy-paste this section from the other proposals.